CHANGELOG_UNRELEASED.md

```diff
@@ -211,6 +211,7 @@ in `qfpoly.v`
     notation for compatibility
   + implicits of `LRMorphism.type`, use the `{lrmorphism _ -> _}`
     notation for compatibility
+  + fixed (generalized) the `comSemiRingType` instance for `prod`
```

Comment:
I'd rather put that in `Added`
```
  + `comSemiRingType` instance for `prod`
```

Reply:
Technically, `comSemiRingType` instance for `prod` was there before this change.

Reply:
Indeed you're right, forget my comment.